haystack/preview/components/readers/extractive.py

```diff
@@ -6,7 +6,9 @@
 from haystack.preview import component, default_from_dict, default_to_dict, ComponentError, Document, ExtractedAnswer
 from haystack.preview.lazy_imports import LazyImport
 
-with LazyImport(message="Run 'pip install farm-haystack[inference]'") as torch_and_transformers_import:
+with LazyImport(
+    "Run 'pip install transformers[torch,sentencepiece]==4.32.1 sentence-transformers>=2.2.0'"
```

Comment:
I don't think we can afford to maintain pinned versions in the error messages, is that required? If it is, I think we could go with one of these:
- point users somewhere in the docs
- make it part of the core deps
- (alas) create an extra in our package

Reply:
I was thinking this could be a temporary solution until we decide how to manage these dependencies better. This pair is quite specific and might stay specific for the foreseeable future, so I think it's worth an extra on its own: but I wouldn't create it now because in `farm-haystack` that would become a bit of a mess.

Wdyt?

Reply:
`extra` is the best option here for the future when we have several separate extras for 2.0. These dependencies are too large for the core deps and not required for all use cases. In order to move fast, I'd say we take the risk of additional maintenance and keep the temporary solution that Sara proposed.